genref/config.yaml

```diff
@@ -44,18 +44,29 @@ apis:
     # Skip this because not referenced anywhere and it has no group name
     skip: true
 
+  - name: kubelet-credentialprovider
+    title: Kubelet CredentialProvider (v1alpha1)
```

Comment:
At release time, how do you know what to add to config.yaml?

Reply:
I'm not aware of any solution that can automate the discovery of new versions.

Reply:
OK. I am trying to sort out how we can provide instructions for the release team (and others) to generate these reference pages.